src/OpenAI/Response/Response.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace PhpLlm\LlmChain\Response;
+namespace PhpLlm\LlmChain\OpenAI\Response;
```

Comment:
was it intentional to move the class? in my understanding that leads to design violations in classes like the `ToolBox` or the `LanguageModel` interface

Reply:
No. It was not intentional. I do not know whta i have mesed up locally but surely in the end i will do a clean up. Was just a fast push to show a proposal and have a discussion about the initial topic. Sorry for the change!